test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -477,27 +467,29 @@ private static void generateWither(ClassWriter cw, String[] nameAndSigValue, Str
 			mv.visitMethodInsn(INVOKEVIRTUAL, "java/lang/Float", "floatValue", "()F", false);
```

Comment:
theres a type with integer above, `java/lang/Interger`

Reply:
it is in some other functions as well, just do a grep for it

Reply:
All instances of `Interger` have been corrected